README.md

```diff
@@ -2,3 +2,91 @@
 
 Automation tools for deploying blockchain networks
 
+## Prerequisites
+Prior to using this script, it's required that docker, sbt, and openjdk are installed on the host.  
+In order to install Docker, please execute the following commands:
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo apt-key fingerprint 0EBFCD88
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io -y
+
+Depending on the use case, it might be advisable, though not required to add the user executing these commands to the docker group. 
+
+In order to install sbt and openjdk, please execute the following commands:
+
+sudo apt-get install -y openjdk-8-jdk-headless
+echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
+sudo apt-get update
+sudo apt-get install -y sbt
+
+
+## Initialize Environment
+
+This script builds and deploys a conseil, postgres, and/or tezos container.  Please note, in order to run the containers, ports 1337(conseil front end), 8732(tezos rpc), and 9732(tezos network)  will need to be opened.  
+
+In this repo, there is provided a default configuration for all three containers, this config folder is located in docker/config/local.  These configuration values can be changed, it's advisable, but not required to copy the config folder to another location so that the original can be kept intact for reference purposes.    
+
+The default network for tezos is alphanet, and this setting is stored in https://github.com/Cryptonomic/Nautilus/blob/master/docker/config/local/tezos/tezos_network.txt This text file only contains one word, currently it's "alphanet".  However, it can be changed to mainnet or zeronet.  
+
+The default configuration for conseil and postgres is contained in /path/to/repo/docker/config/local/conseil/conseil.conf.    
+## Run Script
+
+Depending on the use case the following commands will build the images and deploy the containers.  Please note that the name of the containers will depend on the name of the folder containing the config files, in the default case the folder is "local", so the containers will be named conseil-local, postgres-local, and tezos-node-local.  
+
+### Running with default configs
+Build all three containers:
+```bash /path/to/repo/docker/nautilus.sh -a```
+
+Please note, the following flags can be used together or seperately.
```

Comment:
Rather than paste the output of the help option here, it's better to show the user how to get help. That way, this README doesn't have to be constantly kept in sync with the code.